templates/includes/contact-us.html

```diff
@@ -10,141 +10,225 @@ <h2 class="p-modal__title" id="modal-title">Get your Kubeflow stack running</h2>
         <h3 class="p-heading--5">Have we missed something?</h3>
-        <textarea id="we-missed-something" name="we-missed-something" aria-label="Anything extra you’d like to communicate about your needs or interests?" rows="3"></textarea>
-      </div>
-      <div class="pagination">
-        <a class="pagination__link--previous p-button" href="">Previous</a>
-        <a class="pagination__link--next p-button--positive" href="">Next</a>
+        <textarea id="we-missed-something" name="we-missed-something" aria-label="Anything extra you'd like to communicate about your needs or interests?" rows="3"></textarea>
       </div>
+      <ol class="p-pagination">
+        <li class="p-pagination__item">
+          <a class="p-pagination__link--previous p-button" href="">Previous</a>
+        </li>
+        <li class="p-pagination__item">
+          <a class="p-pagination__link--next p-button--positive" href="">Next</a>
+        </li>
+      </ol>
     </div>
 
     <div class="js-pagination js-pagination--3 u-hide">
       <div class="row u-no-padding">
-        <div class="col-6">
-          <form class="modal-form marketo-form" action="https://pages.ubuntu.com/index.php/leadCapture/save" method="post" id="mktoForm_1337">
+        <div class="col-12">
+          <form action="http://ubuntu.com/marketo/submit" method="post" id="mktoForm_1337">
             <ul class="p-list">
-              <li class="mktFormReq mktField p-list__item">
-                <label for="FirstName" class="mktoLabel">First name:</label>
-                <input required id="FirstName" name="FirstName" maxlength="255" type="text" class="mktoField mktoRequired" required="required" />
+              <li class="p-list__item">
+                <label for="firstName" class="mktoLabel">First name:</label>
+                <input required id="firstName" name="firstName" maxlength="255" type="text" required="required" />
               </li>
-              <li class="mktFormReq mktField p-list__item">
-                <label for="LastName" class="mktoLabel">Last name:</label>
-                <input required id="LastName" name="LastName" maxlength="255" type="text" class="mktoField mktoRequired" required="required" />
+              <li class="p-list__item">
+                <label for="lastName" class="mktoLabel">Last name:</label>
+                <input required id="lastName" name="lastName" maxlength="255" type="text" required="required" />
               </li>
-              <li class="mktFormReq mktField p-list__item">
-                <label for="Email" class="mktoLabel">Email address:</label>
-                <input required id="Email" name="Email" maxlength="255" type="email" class="mktoField mktoEmailField mktoRequired" required="required" />
+              <li class="p-list__item">
+                <label for="email" class="mktoLabel">Email address:</label>
+                <input required id="email" name="email" maxlength="255" type="email" required="required" />
```

Comment:
The pattern needs to be added.

```suggestion
                <input required id="email" name="email" maxlength="255" type="email" required="required" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
```

Reply:
Added